Plan/common/src/main/java/com/djrapitops/plan/delivery/webserver/RequestHandler.java

```diff
@@ -187,9 +189,21 @@ private Request buildRequest(HttpExchange exchange) {
         String requestMethod = exchange.getRequestMethod();
         URIPath path = new URIPath(exchange.getRequestURI().getPath());
         URIQuery query = new URIQuery(exchange.getRequestURI().getRawQuery());
+        byte[] requestBody = new byte[0];
+        try {
+            int b;
+            ByteArrayOutputStream buf = new ByteArrayOutputStream(512);
```

Comment:
OutputStream is not closed after use or if exception occurs. Use try-with-resources to clean up
`try (ByteArrayOutputStream buf = new ByteArrayOutputStream(512)) {`

Reply:
Didn't know that was a thing that already existed by Java 8, thank you